chcemvediet/tests/__init__.py

```diff
@@ -48,6 +49,21 @@ def run_tests(self, *args, **kwargs):
 
 class ChcemvedietTestCaseMixin(TestCase):
 
+    def __init__(self, methodName):
+        super(ChcemvedietTestCaseMixin, self).__init__(methodName)
+        self.patterns_single_column = [
+            u'FROM "accounts_profile" WHERE "accounts_profile"."user_id" = %s LIMIT 21',
```

Comment:
@viliambalaz Tieto stringy nie su sucastou testu. Nijak nezavisia na stave test instancie. Su to len obycajne stringy. Bolo by ich lepsie proste vysunut do nejakeho statickeho suboru v ramci test adresara. Napriklad `chcemvediet/apps/inforequests/tests/render_query_patterns.py` a tam iba hodit globalne:
```
base = [
  ...
]
```
Potom v teste importnut cely `render_query_patterns` a pouzivat `render_query_patterns.base`

Reply:
:ok: